src/torrent/utils/uri_parser.h

```diff
@@ -37,11 +37,6 @@ struct uri_query_state : public uri_base_state {
   uri_query_list elements;
 };
 
-class uri_error : public ::torrent::input_error {
-public:
-  using input_error::input_error;
-};
-
```

Comment:
This needs to be in the header as we want to be able to catch errors by type.

Reply:
Removed.